src/ess/polarization/correction.py

```diff
@@ -208,3 +234,27 @@ def PolarizationAnalysisWorkflow(
         TransmissionFunction[Analyzer]
     ]
     return workflow
+
+
+def HalfPolarizedAnalysisWorkflow(
+    *,
+    polarizer_workflow: sciline.Pipeline,
+) -> sciline.Pipeline:
+    """
+    Create a half-polarized analysis workflow, i.e, with a polarizer but no analyzer.
```

Comment:
Equivalent, for the documentary for later/for users I write "Create a half-polarized workflow"...